src/autoplex/auto/phonons/jobs.py

```diff
@@ -704,33 +653,11 @@ def dft_random_gen_data(
 
     if displacement_maker is None:
         displacement_maker = TightDFTStaticMaker(name="dft rattle static")
```

Comment:
We keep this as we need it in any case, right?

Reply:
I kept this, and in the single atom-displaced supercell part as well, because we are setting different names and there should always be a displacement maker.
![image](https://github.com/user-attachments/assets/5ae83fa6-eadd-4d9a-8c62-96091e668323)